Content.Tests/DMProject/Tests/List/ListNullArg.dm

```diff
@@ -0,0 +1,5 @@
+/proc/ListNullArg2(a[5][3])
+	ASSERT(a[1].len == 3)
+
+/proc/RunTest()
+	ListNullArg2()
```

Comment:
This test errors in BYOND due to `ListNullArg2()`'s argument being null. This needs a `// RUNTIME ERROR` and the top and to be moved back to the broken tests folder.